monitoring/modules_linux_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	. "gopkg.in/check.v1"
 )
 
-type MonitoringSuite struct{}
```

Comment:
I moved this to the DNS test file that doesn't have linux build restrictions. This allows for building/running tests on non-linux platforms.